.env.example

```diff
@@ -0,0 +1,13 @@
+SECRET="Replace with secret string"
+
+DEV_DB_HOST=dbHost # defaults to localhost
+DEV_DB_NAME=nameOfDb
+DEV_DB_USER=nameOfDbUser
+DEV_DB_PASS=dbPassword
+
+TEST_DB_HOST=dbHost # defaults to localhost
```

Comment:
Note as I mentioned my first PR comment that you can specify the same db info for development as for test. I'd recommend against that, but it's up to you!

Reply:
Good point that we can use src/db/queries.js to initialize test data in the future